.github/workflows/test.yml

```diff
@@ -68,14 +70,24 @@ jobs:
       with:
         # Needed to fetch tags, which are required by setuptools-scm.
         fetch-depth: 0
+
+    - name: Download Package
+      uses: actions/download-artifact@v3
+      with:
+        name: Packages
+        path: dist
+
     - name: Set up Python
       uses: actions/setup-python@v2
       with:
         python-version: ${{ matrix.python }}
+
     - name: Install tox
       run: |
         python -m pip install --upgrade pip
         pip install tox
+
     - name: Test
+      shell: bash
       run: |
-        tox -e ${{ matrix.tox_env }}
+        tox run -e ${{ matrix.tox_env }} --installpkg `find dist/*.tar.gz`
```

Comment:
Testing the actual package (alike) we intent to ship is a good pattern, and from the tox output seems to work as intended.

Reply:
at first glance we only test the sdist and not the wheel

Reply:
Indeed, but AFAIK this is how it always has been: tox generates both wheel and sdist, but only installs sdist:

```
λ tox -e py310
...
py310: install_package_deps> python -I -m pip install execnet>=1.1 filelock pytest>=6.2.0
py310: install_package> python -I -m pip install --force-reinstall --no-deps E:\projects\pytest-xdist\.tox\.tmp\package\4\pytest-xdist-3.3.1.dev2+g6d39025.tar.gz
py310: commands[0]> pytest
...
```

Testing both seems a bit overkill, if that's what you are suggesting.